### ISSUE TITLE
InstructionWeavingException on compile, regression since 0.7.0

### ISSUE THREAD

**Issue by Emik03** [oleg-st/InlineMethod.Fody#4]
Updating the nuget package has caused one of my projects to fail to compile, reverting to `0.6.2` works just the same as before.

I'm unable to product a minimal reproducible example right now, but [code that I assume is supposed to be unreachable](https://github.com/oleg-st/InlineMethod.Fody/blob/master/src/InlineMethod.Fody/Extensions/CecilExtensions.cs#L130) was reached, which led to the following compiler error:
```cs
  Fody: An unhandled exception occurred:
Exception:
Unexpected stack-clearing instruction encountered
Type:
InlineMethod.Fody.Helper.InstructionWeavingException
StackTrace:
   at InlineMethod.Fody.Extensions.CecilExtensions.GetPopCount(Instruction instruction) in C:\Work\c#\InlineMethod\src\InlineMethod.Fody\Extensions\CecilExtensions.cs:line 130
   at InlineMethod.Fody.InlineMethodWeaver.AnalyzeArgs() in C:\Work\c#\InlineMethod\src\InlineMethod.Fody\InlineMethodWeaver.cs:line 642
   at InlineMethod.Fody.InlineMethodWeaver.Process() in C:\Work\c#\InlineMethod\src\InlineMethod.Fody\InlineMethodWeaver.cs:line 710
   at InlineMethod.Fody.ModuleWeaver.ProcessCallInstruction(Instruction instruction, MethodDefinition method) in C:\Work\c#\InlineMethod\src\InlineMethod.Fody\ModuleWeaver.cs:line 25
   at InlineMethod.Fody.ModuleWeaver.ProcessMethod(MethodDefinition method) in C:\Work\c#\InlineMethod\src\InlineMethod.Fody\ModuleWeaver.cs:line 45
   at InlineMethod.Fody.ModuleWeaver.Execute() in C:\Work\c#\InlineMethod\src\InlineMethod.Fody\ModuleWeaver.cs:line 67
   at InnerWeaver.ExecuteWeavers() in C:\projects\fody\FodyIsolated\InnerWeaver.cs:line 204
   at InnerWeaver.Execute() in C:\projects\fody\FodyIsolated\InnerWeaver.cs:line 112
Source:
InlineMethod.Fody
TargetSite:
Int32 GetPopCount(Mono.Cecil.Cil.Instruction)
```
The only function that I can think of to trigger this is perhaps [this one](https://github.com/Emik03/Emik.Morsels/blob/bb97565dfaa8ae1780ffde37319ecd12ed0efd1b/Compile/Source/References/Emik/Collections/Specialized/SmallFactory.cs#L63-L85), which must inline the `stackalloc` since otherwise the memory address would be invalid by the time the function returns.

I'm sorry for not providing much information, I hope this is enough to sort the issue out.

**Comment by oleg-st** [comment]
I think it's fixed in 0.7.1
Could you check?

**Comment by Emik03** [comment]
Works wonderfully, thank you for this library!
